gallery/src/components/NxTransferListHalf/NxTransferListHalfPage.tsx

```diff
@@ -69,16 +71,19 @@ const NxTransferListPage = () =>
             <NxTable.Row>
               <NxTable.Cell><NxCode>showMoveAll</NxCode></NxTable.Cell>
               <NxTable.Cell>boolean</NxTable.Cell>
-              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
               <NxTable.Cell></NxTable.Cell>
```

Comment:
Optional props need to have their default value specified when applicable

Reply:
Updated here 9cf90c48d024ff1921bec3ca8088ae74e46aa821